client/src/main/webjar/authentication/authentication.config.js

```diff
@@ -7,9 +7,12 @@
      * Returns the protocol, host, and port part of our single-page-app's URL.
```

Comment:
Please update this line to be accurate (it's doing what the function name says, generating the google oauth callback redirect uri).